lib/galaxy_test/selenium/test_collection_builders.py

```diff
@@ -92,16 +93,15 @@ def test_build_paired_list_hide_original(self):
         second_dataset_reverse = reverse_column.find_elements_by_css_selector("li")[1]
         second_dataset_reverse.click()
 
+        # this toggles the checkbox to not hide originals
         self.collection_builder_hide_originals()
         self.collection_builder_set_name("my awesome paired list")
 
         self.collection_builder_create()
-        self.history_panel_wait_for_hid_ok(3)
```

Comment:
We should add some more assertions here - they seem to work in my testing:

https://gist.github.com/jmchilton/276b8a1deb9763d9acb4fc29c667f229

But we can wait until we merge this and do that in a new PR.

Reply:
fixed it with the prettier re-run :)